docs/index.bs

```diff
@@ -193,15 +193,31 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
         * Detects abnormal operation: such as infinite loops and tasks exceeding imposed time limits (if any) while handling the events.
     </section>
 
-    <section>
+   <section>
       <h4 id="service-worker-events">Events</h4>
 
       The Service Workers specification defines <dfn export id="dfn-service-worker-events">service worker events</dfn> (each of which is an [=event=]) that include (see the <a href="#execution-context-events">list</a>):
           * <dfn export id="dfn-lifecycle-events">Lifecycle events</dfn>: {{install!!event}} and {{activate!!event}}.
           * <dfn export id="dfn-functional-events">Functional events</dfn>: {{fetch!!event}} and the [=events=] defined by other specifications that <a href="#extensibility">extend</a> the Service Workers specification. (See the <a href="#execution-context-events">list</a>.)
           * {{message!!event}} and {{messageerror!!event}}.
     </section>
-  </section>
+    <section>
+      <h4 id="service-worker-timing">Timing</h4>
+
+      Service workers mark certain points in times, that are later exposed by the {{PerformanceNavigationTiming|navigation timing}} API.
+
+      A [=/response=] has an associated null or [=service worker timing info=] <dfn export for=response>service worker timing</dfn>, initially set to null.
+
+      <p>A <dfn export>service worker timing info</dfn> is a <a for=/>struct</a>. It has the
+      following <a for=struct>items</a>:
+
+      <dl>
+      <dt><dfn export for="service worker timing info">start time</dfn> (default 0)
+      <dt><dfn export for="service worker timing info">fetch event dispatch time</dfn> (default 0)
+      <dd>A {{DOMHighResTimeStamp}}.
+      </dl>
```

Comment:
```suggestion
      <section dfn-for="service worker timing info">
        : <dfn export>start time</dfn>
        :: A {{DOMHighResTimeStamp}}, initially 0.
        : <dfn export>fetch event dispatch time</dfn>
        :: A {{DOMHighResTimeStamp}}, initially 0.
      </section>
```

I haven't fully tested the above parses properly, but you see what I'm going for.